Trigger.md

```diff
@@ -0,0 +1,99 @@
+**NOTE:** Kafka bindings are only supported for Azure Functions version 3.x and later versions.
+
+# Examples
+
+# C# Attributes
+
+|Setting|Description|
+|-|-|
+|Topic|Topic Name used for Kafka Trigger|
+|BrokerList|Server Address for kafka broker|
+|ConsumerGroup|Name for the Consumer Group|
+|AvroSchema|Should be used only if a generic record should be generated|
+|LagThreshold|Threshold for lag(Default 1000)|
+
+For connection to a secure Kafka Broker -
+
+|Authentication Setting|librdkafka property|Description|
+|-|-|-|
+|AuthenticationMode|sasl.mechanism|SASL mechanism to use for authentication|
+|Username|sasl.username|SASL username for use with the PLAIN and SASL-SCRAM|
+|Password|sasl.password|SASL password for use with the PLAIN and SASL-SCRAM|
+|Protocol|security.protocol|Security protocol used to communicate with brokers|
+|SslKeyLocation|ssl.key.location|Path to client's private key (PEM) used for authentication|
+|SslKeyPassword|ssl.key.password|Password for client's certificate|
+|SslCertificateLocation|ssl.certificate.location|Path to client's certificate|
+|SslCaLocation|ssl.ca.location|Path to CA certificate file for verifying the broker's certificate|
+
+# Java Annotations
+|Parameter|Description|
+|-|-|
+|name|The variable name used in function code for the request or request body.|
+|topic|Defines the topic.|
+|brokerList|Defines the broker list.|
+|consumerGroup|Name for the Consumer Group.|
+|cardinality|Cardinality of the trigger input. Choose 'One' if the input is a single message or 'Many' if the input is an array of messages. If you choose 'Many', please set a dataType. Default: 'One'|
+|dataType| <p>Defines how Functions runtime should treat the parameter value. Possible values are:</p><ul><li>""(Default): Get the value as a string, and try to deserialize to actual parameter type like POJO.</li><li>string: Always get the value as a string</li><li>binary: Get the value as a binary data, and try to deserialize to actual parameter type byte[].</li></ul>|
+|avroSchema|Avro schema for generic record deserialization|
+
+For connection to a secure Kafka Broker -
+
+|Authentication Setting|librdkafka property|Description|
+|-|-|-|
+|authenticationMode|sasl.mechanism|SASL mechanism to use for authentication|
+|username|sasl.username|SASL username for use with the PLAIN and SASL-SCRAM|
+|password|sasl.password|SASL password for use with the PLAIN and SASL-SCRAM|
+|protocol|security.protocol|Security protocol used to communicate with brokers|
+|sslKeyLocation|ssl.key.location|Path to client's private key (PEM) used for authentication|
+|sslKeyPassword|ssl.key.password|Password for client's certificate|
+|sslCertificateLocation|ssl.certificate.location|Path to client's certificate|
+|sslCaLocation|ssl.ca.location|Path to CA certificate file for verifying the broker's certificate|
+
+# JS/TS/PS/Python Configuration
+|function.json property|Descirption|
+|-|-|
+|type|Must be set to kafkaTrigger.|
```

Comment:
A small 1 or 2 liner for a functions.json will make easy for end-user to understand